runtime/appruntime/apisdk/api/call_meta.go

```
@@ -0,0 +1,216 @@
+		// so we'll just use the first bundled service as the sending service
+		meta.Internal = &InternalCallMeta{
+			SendingService: s.static.BundledServices[0],
+		}
+	}
+
+	return meta
+}
+
+// AddToRequest adds the metadata to the given request
+func (meta CallMeta) AddToRequest(req transport.Transport) {
+	// Future proofing: if we ever create a breaking change to the transport meta
+	// we can use this version number to indicate which version of the meta we're using
+	req.SetMeta("Version", "1")
+
+	// If we're tracing, pass the trace ID, span ID and event ID to the downstream service
+	if !meta.TraceID.IsZero() {
+		// Encode Encore's trace ID and span ID as the traceparent header
+		traceID := hex.EncodeToString(meta.TraceID[:])
+		spanID := hex.EncodeToString(meta.ParentSpanID[:])
+		req.SetMeta(transport.TraceParentKey, fmt.Sprintf("00-%s-%s-01", traceID, spanID))
+
+		// Because Encore does not count an RPC call as a span, but rather a set of events within a span
+		// we also need to pass the event ID which started the RPC call in the tracestate header
+		eventID := strconv.FormatUint(uint64(meta.ParentEventID), 36)
+		req.SetMeta(transport.TraceStateKey, fmt.Sprintf("%s=%s", eventTraceStateKey, eventID))
+	}
+
+	// Pass the correlation ID to the downstream service.
+	// However, we do _not_ pass the X-Request-ID down, as it is not meant to be propagated through request chains
+	if meta.CorrelationID != "" {
+		req.SetMeta(transport.CorrelationIDKey, meta.CorrelationID)
+	}
+
+	// If we're making an internal call, add the internal metadata to the request
+	if meta.Internal != nil {
+		// Add a marker to the request to indicate that this is an internal call
+		req.SetMeta("Internal-Call", meta.Internal.SendingService)
+
+		// TODO(domblack): Add the auth UID and data to the request
+
+		// If we're making an internal call, sign the request
+		if err := svcauth.Sign(svcauth.Noop, req); err != nil {
+
+		}
```

This is a bit weird; use `_ = ...` if we want to deliberately ignore the error (or is the if statement not filled in?)